README.md

```diff
@@ -151,6 +163,12 @@ This means that the built ROM isn't the same as the base one, so something went
 
 Both of these have the disadvantage that the ordering of the terminal output is scrambled, so for debugging it is best to stick to one thread (i.e. not pass `-j` or `-jN`).
 
+## Other versions
+
+To build another version of the game, place the input ROM in `baseroms/<version>/baserom.z64`, and append `VERSION=<version>` to all `make` commands.
+For example, to build `ntsc-1.0`, place a copy of the NTSC 1.0 ROM in `baseroms/ntsc-1.0/baserom.z64`,
+and run `make setup VERSION=ntsc-1.0` and `make VERSION=ntsc-1.0` to build the ROM.
+
```

Comment:
I think its fine to mention it in only one place. I would mention the multiple options you have for doing this though. Maybe not this exact wording but something along these lines
```
To build another version of the game, place the input ROM in `baseroms/<version>/baserom.z64`.
For example, to build `ntsc-1.0`, place a copy of the NTSC 1.0 ROM in `baseroms/ntsc-1.0/baserom.z64`.
Then, there are multiple options for specifying which version gets built:

- run `make setup VERSION=ntsc-1.0` and `make VERSION=ntsc-1.0`
- Modify the VERSION variable directly in the makefile then run `make setup` and `make`
- Create a file called `.make_optoins.mk` and add a line to define the version VERSION=ntsc-1.0
  This has the advantage of setting your own default version locally without modifying the makefile.
  The `.make_optoins.mk` is gitignored by default.
```

Reply:
It felt a bit weird writing about `.make_options.mk` here since it's not really specific to multiversion. Instead, I tried to spruce up the documentation in the Makefile and directed users to there (with a section called "Changing build options" in the README instead of "Other versions"). What do you think? I'm not sure if it deserves more text in the README itself.